sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/InstrumentSelector.java

```diff
@@ -36,18 +39,37 @@ public static Builder builder() {
   public abstract InstrumentType getInstrumentType();
 
   /**
-   * Returns the {@link Pattern} generated by the provided {@code regex} in the {@link Builder}, or
-   * {@code Pattern.compile(".*")} if none was specified.
+   * Returns the {@link Predicate} for filtering instruments by name. Matches everything by default.
    */
-  public abstract Pattern getInstrumentNamePattern();
+  public abstract Predicate<String> getInstrumentNameFilter();
+
+  /** Returns the selections criteria for {@link io.opentelemetry.api.metrics.Meter}s. */
+  public abstract MeterSelector getMeterSelector();
 
   /** Builder for {@link InstrumentSelector} instances. */
   @AutoValue.Builder
   public abstract static class Builder {
     /** Sets a specifier for {@link InstrumentType}. */
     public abstract Builder setInstrumentType(InstrumentType instrumentType);
 
-    abstract Builder setInstrumentNamePattern(Pattern instrumentNamePattern);
+    /** Sets the {@link Pattern} for instrument names that will be selected. */
+    abstract Builder setInstrumentNameFilter(Predicate<String> instrumentNameFilter);
+
+    /** Sets the {@link Pattern} for instrument names that will be selected. */
+    public final Builder setInstrumentNamePattern(Pattern instrumentNamePattern) {
+      return setInstrumentNameFilter(StringPredicates.regex(instrumentNamePattern));
+    }
+
+    /** Sets the exact instrument name that will be selected. */
+    public final Builder setInstrumentName(String instrumentName) {
+      return setInstrumentNameFilter(StringPredicates.exact(instrumentName));
+    }
+
+    /**
+     * Sets the {@link MeterSelector} for which {@link io.opentelemetry.api.metrics.View}s will be
```

Comment:
```suggestion
     * Sets the {@link MeterSelector} for which {@link io.opentelemetry.sdk.metrics.view.View}s will be
```

Reply:
That was a huge typo, should have been "Meter", not "View". Fixed.